WcaOnRails/app/views/registrations/edit.html.erb

```diff
@@ -33,14 +23,27 @@
 
     <%= f.input :comments %>
 
-    <%= f.input :status, as: :radio_buttons, collection: [:a, :p], checked: @registration.accepted? ? :a : :p, include_blank: false %>
+    <%= f.input :status, as: :radio_buttons, collection: [:accepted, :pending, :deleted], checked: @registration.checked_status, include_blank: false %>
 
     <div class="form-group">
       <div class="col-sm-offset-2 col-sm-10">
         <%= f.submit class: "btn btn-default" %>
-        <%= link_to t('registrations.delete'), registration_path(@registration), method: :delete, class: "btn btn-danger", data: { confirm: t('registrations.admin_delete_confirm', name: @registration.name), toggle: "tooltip" }, title: t('registrations.delete_tooltip', name: @registration.name, mail: @registration.email) %>
       </div>
     </div>
+    <hr />
+    <div>
+      <h4>Registration History</h4>
+      Created on <%= wca_local_time(@registration.created_at) %> by <%= @registration.user.name %>
+    </div>
+    <% if @registration.accepted_at %>
+      <div>
+        Accepted on <%= wca_local_time(@registration.accepted_at) %> by <%= @registration.accepted_user.name %>
```

Comment:
Is it possible that this will run into a nil pointer here? It's possible that `@registration.accepted_user` is nil, right?